bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookKeeperTest.java

```diff
@@ -1127,4 +1127,193 @@ public void testBookieAddressResolverPassedToDNSToSwitchMapping() throws Excepti
         }
     }
 
+    @Test
+    public void testCreateLedgerWithIllegalArguments() throws Exception {
```

Comment:
You can use data provider to define the different case of the test.

Reply:
+1